react/features/virtual-background/functions.ts

```diff
@@ -18,6 +18,16 @@ export function checkBlurSupport() {
     return filterSupport;
 }
 
+/**
+ * Checks if virtual background is enabled.
+ *
+ * @returns {boolean} True if virtual background is enabled and false if virtual background is disabled.
+ */
+export function checkVirtualBackgroundEnabled(): boolean {
+
+    return APP.store.getState()['features/base/config'].disableVirtualBackground !== true;
```

Comment:
Can you drop the APP.store usage please, this is what I mean by breaking mobile ... although this feature is not used on mobile at the moment and will not break it, we try to avoid using APP.... as it will go away at some point.

Pass the state to the function the same way as `isScreenVideoShared(state)` is doing it.

Reply:
Thanks, I tough it was because of the npm lint.